nix/.stack.nix/cardano-shell.nix

```diff
@@ -34,6 +34,8 @@
           (hsPkgs.text)
           (hsPkgs.transformers)
           (hsPkgs.turtle)
+          (hsPkgs.yaml)
+          (hsPkgs.time-units)
```

Comment:
When I add this library `time-units`, and run `nix-shell`, it fails with the message
`error: attribute 'time-units' missing, at /home/hiroto/haskell/cardano-shell/nix/.stack.nix/cardano-shell.nix:38:12`

Reply:
Fixed!